### PR TITLE
Add com.w1hkj.flrig

### PR DESCRIPTION
**Pull request by lu9dce**
Transceiver Control
FLRIG is a transceiver control program designed to be used either stand alone or as an adjunct to FLDIGI. The supported transceivers all have some degree of CAT. The FLRIG user interface changes to accommodate the degree of CAT support available for the transceiver in use.

Three different main dialog aspect ratios can be selected to suit the computer screen dimensions and operator preferences. The wide aspect ratio can be resized horizontally. The narrow aspect ratios are fixed in width and height.

Elecraft | Icom | Kenwood | Ten-Tec | Yaesu | Other

IC7100-ui-wide

Transceiver - select transceiver and configure serial i/o parameters
tcpip - configure interface to a remote tcpip/serial controlled transceiver
PTT - configure PTT
AUX - configure separate auxiliary serial ports (if used)
Polling - select and configure transceiver parameters to poll
Trace - select and display program execution paths
Restore - select and configure transceiver parameters to read and restore

### CODE REVIEW

**Comment by lu9dce** [comment]
bot, build com.w1hkj.flrig

File: com.w1hkj.flrig.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ <id>com.w1hkj.flrig</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>GPL-3.0</project_license>
+ <name>flrig</name>
+ <summary>Amateur Radio Rig Control</summary>
+ <developer_name>ARRL, the national association for Amateur Radio</developer_name>
+ <url type="homepage">http://www.arrl.org</url>
+ <description>
+  <p>
+  rig control program, cooperates with fldigi
```
Comment (AsciiWolf) [inline com.w1hkj.flrig.metainfo.xml:13]: Better description from Fedora:
`Flrig is a transceiver control program designed to be used either stand alone or as an adjunct to fldigi.`

File: com.w1hkj.flrig.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 hamradio lu9dce -->
+<component type="desktop-application">
+ <id>com.w1hkj.flrig</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>GPL-3.0</project_license>
+ <name>flrig</name>
```
Comment (AsciiWolf) [inline com.w1hkj.flrig.metainfo.xml:7]: Flrig

(uppercase first letter)

File: com.w1hkj.flrig.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ <summary>Amateur Radio Rig Control</summary>
+ <developer_name>ARRL, the national association for Amateur Radio</developer_name>
```
Comment (AsciiWolf) [inline com.w1hkj.flrig.metainfo.xml:9]: Isn't it developed by http://www.w1hkj.com instead of ARRL?

Comment (lu9dce) [reply com.w1hkj.flrig.metainfo.xml:9]: ok in a while I check everything

File: com.w1hkj.flrig.metainfo.xml (hunk 2 above)
Comment (lu9dce) [reply com.w1hkj.flrig.metainfo.xml:7]: @AsciiWolf 4 o 7?

**Comment by lu9dce** [comment]
bot, build com.w1hkj.flrig

File: com.w1hkj.flrig.metainfo.xml (hunk 2 above)
Comment (AsciiWolf) [reply com.w1hkj.flrig.metainfo.xml:7]: ?

Comment (AsciiWolf) [reply com.w1hkj.flrig.metainfo.xml:7]: Just change the name to **F**lrig.

**Comment by barthalion** [comment]
@AsciiWolf Would you mind handling reaching out to upstream if they'd like to co-maintain their apps on Flathub? Other than this, I guess both flrig and fldigi are ready to merge.

**Comment by lu9dce** [comment]
@barthalion 
I spoke with the developers but none of them answered me
most are radio amateurs
and older people who do not give much importance
do your pr they publish it and give the source code
and anything else they don't care

there are 2 projects one wsjtz and dude-star
which wsjtz told me to take care of
and dude-star to wait ... he'll see what flatpak is about
because I did not know him and later when he is more stable
He was going to let me know .. wsjtz is one of the most important radio software
and dudestar is a digital audio program that replaces a
radio equipment which is very expensive (I mean equipment)
This boy can make a soft that replaces the hard

maybe someone will see him walking in flatpak and join

**Comment by AsciiWolf** [comment]
> @AsciiWolf Would you mind handling reaching out to upstream if they'd like to co-maintain their apps on Flathub? Other than this, I guess both flrig and fldigi are ready to merge.

I think that I already tried asking if they are interested in the Flatpak version of Fldigi (and other tools) and got no reply from them. But it was year ago.

File: com.w1hkj.flrig.metainfo.xml (hunk 2 above)
Comment (barthalion) [reply com.w1hkj.flrig.metainfo.xml:7]: Looks like it's lowercase also in documentation so let's stick to it.

Comment (AsciiWolf) [reply com.w1hkj.flrig.metainfo.xml:7]: It's both lowercase and uppercase. However, it's uppercase in both Fedora and Ubuntu AppData. Flrig is the same.

**Pull request closed without merging**